cli/src/config.rs

```diff
@@ -380,6 +380,7 @@ pub struct Config {
 pub struct ToolchainConfig {
     pub anchor_version: Option<String>,
     pub solana_version: Option<String>,
+    pub package_manager: PackageManager,
```

Comment:
`package_manager` needs to be an optional field like the other ones to keep the manifest backwards-compatible.
```suggestion
    pub package_manager: Option<PackageManager>,
```